packages/block-library/src/style.scss

```diff
@@ -3,6 +3,7 @@
 @import "./block/indicator/style.scss";
 @import "./button/style.scss";
 @import "./categories/style.scss";
+@import "./archives/style.scss";
```

Comment:
Whoops! I'll fix this :)

Reply:
Also in response to your question about the screenshots, I guess the confusion is from me not updating the first two screen shots. The ones that follow are up to date, I left those in there just in case someone wanted to see what it looked like originally.